jerry-core/ecma/operations/ecma-shared-arraybuffer-object.c

```diff
@@ -166,15 +168,34 @@ ecma_op_create_shared_arraybuffer_object (const ecma_value_t *arguments_list_p,
  * @return true - if value is a SharedArrayBuffer object
  *         false - otherwise
  */
-bool
+extern inline bool JERRY_ATTR_ALWAYS_INLINE
 ecma_is_shared_arraybuffer (ecma_value_t target) /**< the target value */
 {
+#if JERRY_BUILTIN_SHAREDARRAYBUFFER
   return (ecma_is_value_object (target)
           && ecma_object_class_is (ecma_get_object_from_value (target), ECMA_OBJECT_CLASS_SHARED_ARRAY_BUFFER));
+#endif /* JERRY_BUILTIN_SHAREDARRAYBUFFER */
+  JERRY_UNUSED (target);
+  return false;
 } /* ecma_is_shared_arraybuffer */
 
+/**
+ * Helper function: check if the target is SharedArrayBuffer Object
+ *
+ * @return true - if value is a SharedArrayBuffer object
+ *         false - otherwise
+ */
+extern inline bool JERRY_ATTR_ALWAYS_INLINE
+ecma_is_shared_arraybuffer_object (ecma_object_t *object_p) /**< the target object */
```

Comment:
Was wondering what our naming scheme is. If I remember correctly if the argument is an object_t then `ecma_object` is the function prefix. So this method should be `ecma_object_is_shared_arraybuffer` (another example: `ecma_object_is_typedarray`)